frontend/views/components/tabs/TabWrapper.vue

```diff
@@ -70,9 +70,24 @@ export default {
       if (this.tabItems.length) {
         this.tabItems[this.activeTab].isActive = true
       }
+    },
+    '$route' (to, from) {
+      const section = to.query.section
+      if (!section) return
+
+      let foundTheTab = false
+      for (const tabItem of this.tabNav) {
+        for (const link of tabItem.links) {
+          if (this.activeTab !== link.index && link.url === section) {
+            this.changeTab(link.index)
+            foundTheTab = true
```

Comment:
This `foundTheTab` variable doesn't appear to be used to do anything except to break out of both loops. Sicne there's no code after the `for` loop, you can delete the `foundTheTab` variable and instead just do `return` on this line.

Reply:
but if we return in the inside line, the outer loop will keep running for nothing. That's the purpose of `foundTheTab`. To break from the outer loop too.

Reply:
... that's not how `return` works. It will break out of the function, including both loops :)

Reply:
Oh... you're right, changed!